addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover.xml

```diff
@@ -2,8 +2,8 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ActivityMarkDonePopover" owl="1">
-        <div class="o_ActivityMarkDonePopover" t-attf-class="{{ className }}" t-on-keydown="activityMarkDonePopoverView and activityMarkDonePopoverView.onKeydown" t-ref="root">
-            <t t-if="activityMarkDonePopoverView">
+        <t t-if="activityMarkDonePopoverView">
+            <div class="o_ActivityMarkDonePopover" t-attf-class="{{ className }}" t-on-keydown="activityMarkDonePopoverView and activityMarkDonePopoverView.onKeydown" t-ref="root">
```

Comment:
is `activityMarkDonePopoverView and` still needed now that this node is guarded?

Reply:
Good catch, note that it's the main reason to do this change.

Owl 2 does not support `undefined` event handler, so previous guard is useless.

Reply:
Changes done